packages/dialtone-vue2/recipes/conversation_view/editor/editor.vue

```diff
@@ -41,7 +42,7 @@
                   class="d-fw-bold"
```

Comment:
This doesn't do anything. Remove?

Reply:
doesn't look like it does. will remove this.